src/Input.js

```diff
@@ -5,7 +5,7 @@ import deprecationWarning from './utils/deprecationWarning';
 
 class Input extends InputBase {
   render() {
-    if (this.props.type === 'static') { // eslint-disable-line react/prop-types
+    if (this.props.type === 'static') {
```

Comment:
Sorry about my `two branches` nagging... It is needed and the right thing. I know.
I was very tired.

--
I know the problem is not in `two branches` but in that we put too much changes to `v0.24` at once.